[Bug Report] Panic in Generate

Probably introduced in #2167

```
panic: runtime error: integer divide by zero

goroutine 105 [running]:
github.com/stashapp/stash/pkg/manager.(*GeneratorInfo).configure(0xc000136500)
    github.com/stashapp/stash/pkg/manager/generator.go:115 +0x9d
github.com/stashapp/stash/pkg/manager.(*PreviewGenerator).Generate(0xc000018310)
    github.com/stashapp/stash/pkg/manager/generator_preview.go:56 +0x78
github.com/stashapp/stash/pkg/manager.(*GeneratePreviewTask).Start(0xc00039a000, {0x198fc98, 0xc0001fa600})
    github.com/stashapp/stash/pkg/manager/task_generate_preview.go:59 +0x4b5
github.com/stashapp/stash/pkg/manager.(*GenerateJob).Execute.func3()
    github.com/stashapp/stash/pkg/manager/task_generate.go:127 +0x3c
github.com/stashapp/stash/pkg/job.(*Progress).ExecuteTask(0xc0001fa700, {0xc00038e160, 0xa9}, 0xc0001f2d00)
    github.com/stashapp/stash/pkg/job/progress.go:153 +0xcd
created by github.com/stashapp/stash/pkg/manager.(*GenerateJob).Execute
    github.com/stashapp/stash/pkg/manager/task_generate.go:126 +0x3bb
```

**Stash Version: (from Settings -> About):**
0.12.0

**Desktop (please complete the following information):**
 - OS: Docker on MacOS

--- a/pkg/manager/generator.go
+++ b/pkg/manager/generator.go
@@ -114,6 +114,15 @@ func (g *GeneratorInfo) configure() error {
 		return err
 	}
 
+	// #2250 - ensure ChunkCount and ChunkDuration are valid
+	if g.ChunkCount < 1 {
+		logger.Warnf("[generator] Segment count (%d) must be > 0. Using 1 instead.", g.ChunkCount)
+		g.ChunkCount = 1
+	}
+	if g.ChunkDuration < 0.75 {
+		logger.Warnf("[generator] Segment duration (%f) must be >= 0.75. Using 0.75 instead.", g.ChunkDuration)
+		g.ChunkDuration = 0.75
+	}
 	g.NthFrame = g.NumberOfFrames / g.ChunkCount
 
 	return nil

--- a/ui/v2.5/src/components/Settings/GeneratePreviewOptions.tsx
+++ b/ui/v2.5/src/components/Settings/GeneratePreviewOptions.tsx
@@ -47,11 +47,12 @@ export const VideoPreviewInput: React.FC<IVideoPreviewInput> = ({
         <Form.Control
           className="text-input"
           type="number"
-          value={previewSegments?.toString() ?? 0}
+          value={previewSegments?.toString() ?? 1}
+          min={1}
           onChange={(e: React.ChangeEvent<HTMLInputElement>) =>
             set({
               previewSegments: Number.parseInt(
-                e.currentTarget.value || "0",
+                e.currentTarget.value || "1",
                 10
               ),
             })

